.travis.yml

```diff
@@ -1,7 +1,23 @@
-sudo: required
 services: [ docker ]
+stages:
+  - build-image
+  - name: trigger-dockerhub
+    if: branch = master AND type != pull_request
```

Comment:
[CI docs](https://docs.travis-ci.com/user/conditions-v1) state that `branch` env, stores current branch name; for pull requests: the base branch name.